db/schema.rb

```diff
@@ -10,7 +10,7 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema[7.0].define(version: 2023_08_25_140331) do
+ActiveRecord::Schema[7.0].define(version: 2024_02_17_054158) do
```

Comment:
I think this is mistakenly added?

Reply:
Actually I think after adding mentor_token and other fields the version is changed in my local system.